pkg/bbgo/strategy_controller.go

```diff
@@ -0,0 +1,79 @@
+package bbgo
+
+import (
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+type StrategyController struct {
+	Status types.StrategyStatus
+
+	// Callbacks
+	SuspendCallback       func() error
```

Comment:
name it as `suspendCallbacks []func() error` then you can use callbackgen

Reply:
done